src/Persistence/YamlValidator/ValidationProcessor.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.YamlValidator;
+
+public class ValidationProcessor
+{
+    private readonly YamlLoader _fileLoader;
+    private readonly SchemaLoader _schemaLoader;
+    private readonly Validator _validator;
+
+    public ValidationProcessor(YamlLoader fileLoader, SchemaLoader schemaLoader, Validator validator)
```

Comment:
should this use the new IValidator interface?

Reply:
This file was intended to be deleted. These are results of the merge conflict (which should now be fixed). I wanted to re-request review after I fixed the merge conflict so the PR wouldn't be confusing 😞